packages/lib/test/src/ProxyAdmin.test.js

```diff
@@ -11,8 +11,8 @@ const ImplV1 = Contracts.getFromLocal('DummyImplementation');
 const ImplV2 = Contracts.getFromLocal('DummyImplementationV2');
 const ProxyAdminContract = Contracts.getFromLocal('ProxyAdmin');
 
-contract('ProxyAdmin class', function(accounts) {
-  const [_, proxyAdminOwner, newAdmin, otherAccount] = accounts.map(utils.toChecksumAddress);
+contract.only('ProxyAdmin class', function(accounts) {
```

Comment:
Watch out! By the way, we need to add something that makes the CI fail if there is a `.only` in the specs. Or a git hook that runs on commit and checks it.

Reply:
Would great to have cli to catch `.only`. Funny enough I was thinking the same.